roles/openshift_master/templates/v1_partials/oauthConfig.j2

```diff
@@ -77,6 +77,8 @@ oauthConfig:
   - name: {{ identity_provider.name }}
     challenge: {{ identity_provider.challenge }}
     login: {{ identity_provider.login }}
+{% set default_mapping_method = 'add' if openshift.master.identity_providers | length > 1 else 'claim' %}
+    mappingMethod: {{ identity_provider.mappingMethod | default(default_mapping_method) }}
```

Comment:
@liggitt does this look like a reasonable default to you? When the number of identity providers specified is > 1 we default to using add as the mappingMethod, if only one is specified we use claim as the default, with the ability to override the default by specifying the mappingMethod in the identity provider definition.